<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IOT_NSL</class>
 <widget class="QWidget" name="IOT_NSL">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
    </font>
   </property>
   <property name="text">
    <string>开始监测</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>20</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>停止监测</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>泥石流监测</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
    </font>
   </property>
   <property name="text">
    <string>湿度</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="hum_lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>511</width>
     <height>211</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
    </font>
   </property>
   <property name="title">
    <string>湿度实时数据</string>
   </property>
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>491</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>20</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>511</width>
     <height>211</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
    </font>
   </property>
   <property name="title">
    <string>降雨量实时数据</string>
   </property>
   <widget class="QCustomPlot" name="customPlot_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>491</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="pre_lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
    </font>
   </property>
   <property name="text">
    <string>降雨量</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="warning_NSL" native="true">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>90</y>
     <width>211</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>340</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1:湿度超过限值次数</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>380</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2:降雨量超过限值次数</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>420</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3:预警次数次数</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/b1.jpg);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>startButton</zorder>
  <zorder>stopButton</zorder>
  <zorder>label</zorder>
  <zorder>label_3</zorder>
  <zorder>hum_lineEdit</zorder>
  <zorder>groupBox</zorder>
  <zorder>back</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>pre_lineEdit</zorder>
  <zorder>label_4</zorder>
  <zorder>warning_NSL</zorder>
  <zorder>label_2</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
